src/coreclr/vm/prestub.cpp

```diff
@@ -2008,6 +2011,10 @@ extern "C" void* STDCALL ExecuteInterpretedMethod(TransitionBlock* pTransitionBl
     InterpThreadContext *threadContext = GetInterpThreadContext();
     int8_t *sp = threadContext->pStackPointer;
 
+    InterpByteCodeStart* pInterpreterCode = dac_cast<PTR_InterpByteCodeStart>(byteCodeAddr);
+
+    GCX_MAYBE_COOP(((MethodDesc*)pInterpreterCode->Method->methodHnd)->HasUnmanagedCallersOnlyAttribute());
```

Comment:
`HasUnmanagedCallersOnlyAttribute` is fairly expensive check - it reads custom attributes. We do not want to do this check every time we enter interpreter.

Instead, this should be pre-computed in the interpreter bytecode.

Reply:
Also, we should do all this https://github.com/dotnet/runtime/blob/main/src/coreclr/vm/jithelpers.cpp#L2310-L2316 for UnmanagedCallersOnly methods.

Reply:
Ah, I haven't realized this flag is set only for UnmanagedCallersOnly.